core/frontend/src/App.vue

```diff
@@ -211,7 +212,7 @@ export default Vue.extend({
 
   data: () => ({
     settings,
-    drawer: false,
+    drawer: undefined as boolean|undefined,
```

Comment:
What is the reason to `drawer` to be an union and not a simple boolean ?

Reply:
With this patch, the drawer can start as both open or close, depending on the user screen size. As we don't know how it is at start, we make it initial value undefined, and the v-model will take care of updating at mounting time.